cmd/client_openshift.go

```diff
@@ -0,0 +1,23 @@
+//go:build openshift
```

Comment:
@matejvasek sorry for my newbie question.. is this only going to be built if it is an "openshift" build?

Reply:
Tags are no longer in use in latest version.